### PR TITLE
Added Javadocs to login and register use case files, made email_regex…

### PR DESCRIPTION
**Pull request by FFFiend**
… final attribute in register use case.

### CODE REVIEW

File: src/main/java/use_cases/login_leaderboard/LoginUser.java [hunk 1]
```diff
@@ -2,8 +2,6 @@
 
 import entities.login_leaderboard.User;
 
-import javax.swing.*;
-
```
Comment (tseamuscorlett) [inline src/main/java/use_cases/login_leaderboard/LoginUser.java:6]: Awesome. we should check for unused imports/variables/methods in other classes we have!

**Review by tseamuscorlett — APPROVED** [review]

**Review by ltenenb — DISMISSED** [review]
Everything looks good!

File: src/test/java/leaderboard/ControllersTest.java [hunk 2]
```diff
@@ -23,12 +23,13 @@ public void LoginUserControllerTest() {
         loginUseCase.setUsers(FileReader.create().PREV.getUsers());
         LoginUserController logincontroller = new LoginUserController(loginUseCase);
 
-        // Assertions
-//        Assertions.assertEquals("yes", logincontroller.performUseCase("Owais", "Owais.93"));
-//        Assertions.assertEquals("no", logincontroller.performUseCase("abc", "hello"));
-//        Assertions.assertEquals("no", logincontroller.performUseCase("Sean", "hello"));
+
+        Assertions.assertEquals("no",logincontroller.performUseCase("abc","hello"));
```
Comment (tseamuscorlett) [inline src/test/java/leaderboard/ControllersTest.java:27]: this fails Autograding

**Review by tseamuscorlett — CHANGES_REQUESTED** [review]

**Comment by ltenenb** [comment]
I think the problems should be fixed now.
- There was a missing import in ControllersTest.
- When GitHub runs the controllers test, it got an error because it couldn't open the popup window.
Now all the tests should pass...

**Pull request merged**